src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,163 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidUserException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private static RegistrationService registrationService;
+
+    // Константы для повторно используемых значений
+    private static final String VALID_LOGIN = "validLogin";
+    private static final String VALID_PASSWORD = "validPass";
+    private static final String EXISTING_LOGIN = "existingLogin";
+    private static final String PASSWORD1 = "password1";
```

Comment:
```suggestion
    private static final String PASSWORD = "password1";
```

don't use numbers for naming